zanata-common-api/src/main/java/org/zanata/rest/service/AsynchronousProcessResource.java

```diff
@@ -167,6 +167,9 @@ public ProcessStatus startSourceDocCreationOrUpdate(
      *            Auto will check the history of your translations and will not
      *            overwrite any translations for which it detects a previous
      *            value is being pushed.
+     * @param isMyTranslations
+     *            Uploader will take credit of all uploaded translations as
+     *            their own.
```

Comment:
Recommend wording as "The translator field for all uploaded translations will be set to the user who performs the upload."